lib/tessel/commands.js

```diff
@@ -5,7 +5,7 @@ module.exports.scanWiFi = function() {
   return ['ubus', 'call', 'iwinfo', 'scan', '{"device":"wlan0"}'];
 };
 module.exports.stopRunningScript = function() {
-  return ['/etc/init.d/tessel-app', 'stop'];
+  return ['killall', 'node'];
```

Comment:
Seems a little extreme, especially if we ever add something on the system that uses node to run. Is there a way to have push/run return the pid and kill off of that?